### ISSUE TITLE
Meeting Notes 2016-12-19

### ISSUE THREAD

**Issue by lyallg** [TheCanadianConfederationDebates/TCCD#45]
DH: May have more FN data for personography upcoming.
MH: Are they using ids from parliamentarians?
DH: there are a few.

MH: When treaty stuff done?
DH: It is done but may be more coming.
--- pursuing more funding for NWT and Nunavit
--- additions won't be subject to July 1 deadline.

GL: no more mondays starting Jan
DH: flexible.
MH and GL: to decide what new TCCD meeting day.

DH: to send jpg of missing NFLD H of C image.
MH: to ocr
GL: to edit trans and process files

DH: uploaded personography and schema

DH: should transcribers enter data from headers?
GL: yes.
MH: we have a function to retrieve call numbers when entered
GL: plus ST and I manually put original page numbers in file names which get processed in the TEI creation

DH: GF almost done a few packages, expect their delivery
--- she may ask for more

DH: reserves plotting?
--- Is the kml format useful?
MH: "a network link in kml not available"
--- they are links, data doesn't work
DH: BC file works
MH: file looks ok in Oxygen but it is full of network links. What do we want to do with this?
DH: It would be nice tp plot reserves like ridings.
MH: Can use co-ordinates.
DH: can we harvest from this file? Automatic points?

MH: We can be in danger of offending people with inaacurate points.
--- but will do what DH asks from programming pov

DH: Plan B is to say "Treaty 6 signed on this hill" etc. But this will mean 50 some odd people have same point. Maybe not useful?

MH: sends list of place names from kml

DH: JB and his RAs are testing feasibility of this plotting
--- bottom line gov website best as reference point

MH: runs diagnostics

MH: to do web output before mapping

DH: do we need a directory of images for people
MH: not good to embed images from other sites
DH: images are public domain. Someone should gahter them and name by xml id?
MH: a list of images named by xml id
-- will standardize size after
